### ISSUE TITLE
Reference to ANTIALIAS in PIL

### ISSUE THREAD

**Issue by MDCurrent** [nerfstudio-project/nerfstudio#2378]
**Describe the bug**
When running the docker image 

The process data script fails to load the pixel-perfect extractor

**To Reproduce**
1. Run through the docs - when running the interactive docker container
```docker run --gpus all  -u 1000 -v /home/mdcurrent/Videos:/workspace/  -v /home/mdcurrent/.cache/:/home/user/.cache/  -p 7007:7007 --rm -it --shm-size=12gb  nerfstudio-89```

see the error 
```❌ Completion script generation failed: ['ns-process-data', '--tyro-print-completion', 'bash'] install.py:124
           Traceback (most recent call last):                                                             install.py:128
             File "/usr/local/bin/ns-process-data", line 5, in <module>                                                 
               from nerfstudio.scripts.process_data import entrypoint                                                   
             File "/home/user/nerfstudio/nerfstudio/scripts/process_data.py", line 36, in <module>                      
               from nerfstudio.process_data.colmap_converter_to_nerfstudio_dataset import                               
           BaseConverterToNerfstudioDataset                                                                             
             File                                                                                                       
           "/home/user/nerfstudio/nerfstudio/process_data/colmap_converter_to_nerfstudio_dataset.py",                   
           line 21, in <module>                                                                                         
               from nerfstudio.process_data import colmap_utils, hloc_utils, process_data_utils                         
             File "/home/user/nerfstudio/nerfstudio/process_data/hloc_utils.py", line 45, in <module>                   
               from pixsfm.refine_hloc import PixSfM  # type: ignore                                                    
             File "/home/user/pixel-perfect-sfm/pixsfm/__init__.py", line 19, in <module>                               
               from . import (  # noqa F403                                                                             
             File "/home/user/pixel-perfect-sfm/pixsfm/features/__init__.py", line 2, in <module>                       
               from . import models, extractor, extract_patches, store_features, store_references  # noqa               
           F403                                                                                                         
             File "/home/user/pixel-perfect-sfm/pixsfm/features/extractor.py", line 32, in <module>                     
               class FeatureExtractor:                                                                                  
             File "/home/user/pixel-perfect-sfm/pixsfm/features/extractor.py", line 64, in                              
           FeatureExtractor                                                                                             
               filters = {                                                                                              
             File "/home/user/pixel-perfect-sfm/pixsfm/features/extractor.py", line 65, in <dictcomp>                   
               n: getattr(PIL.Image, n)                                                                                 
           AttributeError: module 'PIL.Image' has no attribute 'ANTIALIAS'  
```

**Expected behavior**
Haven't gotten a clean run, but expect the docker container to boot and not die instantly

**Screenshots**
![Screenshot from 2023-08-28 22-07-12](https://github.com/nerfstudio-project/nerfstudio/assets/15005893/2ea1b6c4-8b21-4eb1-9883-f7472b4e64b7)

**Additional context**
i could be missing something but i think i set the right 8.9/ 89 everywhere applicable and updated nvidia drivers / a clean build

Looks like the attribute has been sunset 
https://pillow.readthedocs.io/en/stable/releasenotes/10.0.0.html#constants

TIA!

**Comment by GabrielGosden** [comment]
Hi,

According to [10.0.0 Pillow release notes ](https://pillow.readthedocs.io/en/stable/releasenotes/10.0.0.html#constants) `Image.ANTIALIAS` has been removed and replaced by ` Image.LANCZOS `or `Image.Resampling.LANCZOS`. Therefore built the docker image using version 9.5.0 by adding the following to the Dockerfile as suggested on [Stack Overflow](https://stackoverflow.com/questions/76616042/attributeerror-module-pil-image-has-no-attribute-antialias):

`pip install Pillow==9.5.0`

This fixed the problem for me and I was able launch the docker image without issues. Hope this helps.
